resources/views/vendor/notifications/email.blade.php

```diff
@@ -0,0 +1,62 @@
+@component('mail::message')
+{{-- Greeting --}}
+@if (! empty($greeting))
+# {{ $greeting }}
+@else
+@if ($level === 'error')
+# @lang('Whoops!')
+@else
+# @lang('Hello!')
```

Comment:
Added a question on this being left in or not in the google doc describing the new copy.